plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -1813,6 +1897,49 @@ def exec_module(self, **kwargs):
                     if self.license_type is not None:
                         vm_resource.license_type = self.license_type
 
+                    if self.vm_identity is not None:
+                        # If 'append' is set to True save current user assigned managed identities to use later
+                        if self.vm_identity.get('user_assigned_identities', {}) is not None and self.vm_identity.get('user_assigned_identities', {}).get('append', False) is True:
```

Comment:
line too long (178 > 160 characters)
```suggestion
                        if self.vm_identity.get('user_assigned_identities', {}) is not None and self.vm_identity.get('user_assigned_identities', {}).get('append', False) is True:
```

Reply:
I have splitted this line in two